assets/dev/js/frontend/handlers/video.js

```diff
@@ -133,7 +133,17 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 			return;
 		}
 
-		this.apiProvider.onApiReady( ( apiObject ) => this.prepareYTVideo( apiObject ) );
+		// When Optimized asset loading is set to off, the video type is set to 'Youtube', and 'Privacy Mode' is set
+		// to 'On', there might be a conflict with other videos that are loaded WITHOUT privacy mode, such as a
+		// video bBackground in a section. In these cases, to avoid the conflict, a timeout is added to postpone the
+		// initialization of the Youtube API object.
+		if ( ! elementorFrontend.config.experimentalFeatures[ 'e_optimized_assets_loading' ] && elementSettings.yt_privacy ) {
+			setTimeout( () => {
```

Comment:
You can always defer the loading, even if the feature is on.

Reply:
Done.